backend/migrations/20240313130248_poultry_checker.up.sql

```diff
@@ -0,0 +1,5 @@
+-- Add up migration script here
+ALTER TYPE meal_type add value 'POULTRY';
+COMMIT;
```

Comment:
is this commit necessary/does it do anything?

Reply:
Yes, it ensures that the previous operation is finished, before the next one is started.